modules/openapi-generator/src/test/resources/3_0/typescript/19027-regression.yaml

```diff
@@ -0,0 +1,31 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Sample for uniqueItems
+servers:
+  - url: http://localhost:3000
+paths:
+  /unique-items:
+    get:
+      operationId: unique_items
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
```

Comment:
I wasn't sure whether it is acceptable to just remove it, but this part of the definition is not actually needed to show the issue.